CMSgov/design-system
# [WNMGDS-2107] DMS migration into doc site

## Summary

- Consistent header - Move content and images into doc site page
- Print guidelines - Create page under guidelines section of doc site
- Help drawer - Update to include icon on doc site and update doc site story to include link trigger
- Button content guidance - Add guidance for medicare it's mostly similar to HCgov 
- Utilize utility class to create example on the doc site of a teal color header

## How to test

1. run doc site with `yarn start` and visit the following pages

- Mgov header to see images and additional content
- new print guidelines page under guidelines header
- Updated help drawer that shows link to toggle help drawer and mgov has a logo with the trigger
- Button content guidance added for Mgov
- Updated heading page content to add on dark headings, link headings and custom mgov headings

## Checklist




### If this is a change to code:


### If this is a change to documentation:
